docs/pipelineruns.md

```diff
@@ -97,6 +97,26 @@ of the `TaskRun` resource object.
 For examples and more information about specifying service accounts, see the
 [`ServiceAccount`](./auth.md) reference topic.
 
+## Labels
```

Comment:
This seemed like the least bad place for the docs, but I don't know if it really fits. Any suggestions are welcome!

Reply:
i had trouble thinking of a good place myself! the only other alternative i can think of is to create a new doc like `troubleshooting.md` or `faq.md` or `tips.md` or something but none of those options seem great either 🤔 this seems fine for now tho if you don't like any of those ideas!

Reply:
Yeah, it seems kind of weird to create a new file just for this information. My preference would be to leave it for now, and then if we end up with other troubleshooting-esque docs we can refactor them into a single place later. Does that seem ok?